src/app/tests/suites/certification/ci-pics-values

```diff
@@ -1517,11 +1517,21 @@ DRLK.S.A0031=1
 DRLK.S.A0032=1
 DRLK.S.A0033=1
 DRLK.S.A0035=0
+DRLK.S.A0080=0
+DRLK.S.A0081=0
+DRLK.S.A0082=0
+DRLK.S.A0083=0
+DRLK.S.A0084=0
+DRLK.S.A0085=0
+DRLK.S.A0086=0
+DRLK.S.A0087=0
+DRLK.S.A0088=0
 
 #ReadOnly attributes
 DRLK.S.A0028.ReadOnly=1
 DRLK.S.A002c.ReadOnly=1
```

Comment:
These should be removed, if you're removing them from PICS.yaml, right?

Reply:
once we fix the above one, we don't need to touch this one.